### DISCUSSION TITLE
audio autoplay in a bubblewrap/PWAbuilder app ?

### DISCUSSION THREAD

**Discussion by JAWAwebgames** [GoogleChromeLabs/bubblewrap#849] [Q&A]
Hi,
Is there a way to allow background audio autoplay when launching a pwabuilder app, without requiring a first user gesture ? It's already allowed in the installed PWA (with a TWA set) so I can't figure out why it's not woking with the same PWA embeded into the apk.
Thanks !

**Comment by mirekkukla** [comment]
We're seeing the exact same behavior: when installed manually as a PWA, our app allows audio to play without first requiring user interaction. However, when bubblewrapped and installed natively as a TWA, user interaction is suddenly required before we can play audio.

I'd expect that the same rationale for allowing auto-play in the first scenario (the user chose to actively install the app) applies in the second, so we were rather confused by this divergence in behavior.

This is a big deal for us as we're operating in an enterprise setting, and running our app as a TWA is the only option we have (**our device management solution(*) doesn't support installing PWAs**, but does support remote management of native apps, such as TWAs).

(*) Airwatch by VMware

**Comment by PEConn** [comment]
For reference, the Chrome bug tracking this issue is:
https://issues.chromium.org/issues/40759733
